depends/patches/qt/qt5-tablet-osx.patch

```diff
@@ -0,0 +1,20 @@
+--- old/qtbase/src/widgets/kernel/qwidgetwindow.cpp	2014-09-05 20:45:18.717570370 -0400
```

Comment:
Can you confirm that this patch is still required? It might have been committed to the version of Qt 5 that we'll (eventually) be using.